pkg/k8s/endpoints.go

```diff
@@ -143,7 +143,12 @@ func ParseEndpoints(ep *slim_corev1.Endpoints) (ServiceID, *Endpoints) {
 
 	for _, sub := range ep.Subsets {
 		for _, addr := range sub.Addresses {
-			backend, ok := endpoints.Backends[cmtypes.MustParseAddrCluster(addr.IP)]
+			addrCluster, err := cmtypes.ParseAddrCluster(addr.IP)
+			if err != nil {
+				continue
```

Comment:
Should we log that something weird happened here? Normally this should be pretty unexpected (I think the user who reported this had some external service create these ceps).

Reply:
Same question in all the continues

Reply:
Yep, good point. I wouldn't prefer logging in k8s resource handler because it makes too many logs, but in this case, it's ok. As you said, it's pretty unexpected.

Reply:
Umm, but these are the parsing functions, so I also wouldn't prefer introducing side effects in the parsing function. If we want to report errors to users, we should return multiple errors and generate logs on the caller side.

Reply:
Also, k8s has a validation, so I think we don't have to worry too much. I intentionally put an invalid IP address to the Endpoint object and got this error.

```
# endpoints "echoserver" was not valid:
# * subsets[0].addresses[0].ip: Invalid value: "10.2.0.512": must be a valid IP address, (e.g. 10.9.8.7 or 2001:db8::ffff)
# * subsets[0].addresses[0].ip: Invalid value: "10.2.0.512": must be a valid IP address

Reply:
Makes sense